README.md

```diff
@@ -1,45 +1,117 @@
+
+  - **plugins?: Plugins**
+
+    The `plugins` property is optional and contains plugins for component. For more information on what plugin is, read the [Plugin characteristic](./docs/props/plugin.md) document.
+
+### SubComponents
+
+The list of components available in the `documentation-component` includes:
+
+  - **Provider**
+
+    The `sources` property is required and contains sources for component. For more information on what render engine is, read the [Sources characteristic](./docs/props/sources.md) document.
+
+  - **Content**
+
+    The `renderEngines` property is required (require is only one property in array) and contains render engines for component. For more information on what render engine is, read the [Render engine characteristic](./docs/props/render-engine.md) document.
+
+  - **SourceTypes**
 
-### Launch a development environment
+    The `plugins` property is optional and contains plugins for component. For more information on what plugin is, read the [Plugin characteristic](./docs/props/plugin.md) document.
 
-Launch the development server with the hot reloading functionality that allows you to immediately see any change made in files in the `playground/src` and `packages/documentation-component/src` folders in the browser. Use this command:
+### Example
 
-``` sh
-$ npm start
+See exemplary using below.
+
+``` tsx
+import React from "react";
+import { render } from "react-dom";
+import {
+  DC,
+  Content,
+  Sources,
+  RenderEngines,
+  Plugins,
+} from '@kyma-project/documentation-component';
+import { markdownRenderEngine, plugins as markdownPlugins } from '@kyma-project/dc-markdown-render-engine';
+
+const SOURCES: Sources = [
+  {
+    source: {
+      type: "md",
+      rawContent: "Example content",
+    }
+  }
+]
+
+const RENDER_ENGINES: RenderEngines = [
+  markdownRenderEngine,
+];
+
+const PLUGINS: Plugins = [
+  markdownPlugins.frontmatterMutationPlugin,
+];
+
+const App: React.FunctionComponent<> = () => (
+  <DC.Provider
+    sources={SOURCES}
+    renderEngines={RENDER_ENGINES}
+    plugins={PLUGINS}
+  >
+    <Content />
+  </DC.Provider>
+);
+
+render(<App />, document.getElementById("root"));
 ```
 
-You can access the live development server at [localhost:3000](http://localhost:3000/).
\ No newline at end of file
+### Custom render engine
+
+For information how to write custom render engine for specific types, read [Custom render engine](./docs/guidelines/custom-render-engine.md) document.
+
+### Custom plugin
+
+For information how to write custom plugin for specific types, read [Custom plugin](./docs/guidelines/custom-plugin.md) document.
+
+## Development
+
+For information on how to set up a development environment, write and run tests, follow the naming and architecture convention defined for the project in the [Development Guide](./docs/development/guide.md).
+
+## Contribution
```

Comment:
I would remove this section as well - we already have the `CONTRIBUTING.md` doc at the repo's root and it points to the correct doc.